_layouts/visualisation.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+---
+
+<section class="framework__page">
+	<!-- Page Title -->
```

Comment:
Super minor thing, but we use two-space indents everywhere else, could we do so here too?